File: C# .Net/JDI UI Framework/JDI/Web/Selenium/Elements/Complex/RadioButtons.cs
using Epam.JDI.Core.Interfaces.Complex;
using System;

namespace Epam.JDI.Web.Selenium.Elements.Complex
{
    public class RadioButtons : RadioButtons<IConvertible>, IRadioButtons { }
    public class RadioButtons<TEnum> : Selector<TEnum>, IRadioButtons<TEnum>
        where TEnum : IConvertible
    {
    }
}

File: Python/JDI/web/selenium/elements/complex/radio_buttons.py
from JDI.web.selenium.elements.complex.selector import Selector


class RadioButtons(Selector):

    def __init__(self, by_options_name_locator_template=None, by_all_options_names_locator=None):
        super(RadioButtons, self).__init__(by_options_name_locator_template, by_all_options_names_locator)